src/server/views/confirmation.html

```diff
@@ -10,6 +10,8 @@
     {{ govukPanel({
       titleText: "Application complete"
     }) }}
+    <h2 class="govuk-heading-m">What happens next</h2>
```

Comment:
Can we make sure the "Thank you" line comes first?

Then follow up with "What happens next"

Reply:
Done